tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -1611,6 +1613,69 @@ def test_auto_dtype_with_helix(self, comms_medium, cuda_graph_config,
                                       self.MODEL_PATH) as llm:
             run_accuracy_test(llm, self.MODEL_NAME, ["GSM8K"])
 
+    @pytest.mark.parametrize(
+        "gen_tp_pp", [(1, 1), (1, 2), (2, 1), (2, 2)],
+        ids=["gen_tp1pp1", "gen_tp1pp2", "gen_tp2pp1", "gen_tp2pp2"])
+    @pytest.mark.parametrize("ctx_tp_pp", [(1, 1), (1, 2), (2, 1), (2, 2),
+                                           (1, 4)],
+                             ids=[
+                                 "ctx_tp1pp1", "ctx_tp1pp2", "ctx_tp2pp1",
+                                 "ctx_tp2pp2", "ctx_tp1pp4"
+                             ])
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)